compiler/rustc_mir/src/transform/inline.rs

```diff
@@ -300,9 +301,6 @@ impl Inliner<'tcx> {
         debug!("    final inline threshold = {}", threshold);
 
         // FIXME: Give a bonus to functions with only a single caller
-
-        let param_env = tcx.param_env(callee_body.source.def_id());
```

Comment:
note: this was wrong even if it used the caller source as we have to use a `ParamEnv` with `RevealAll`. This caused an ICE in #77306